nipype/interfaces/dcm2nii.py

```diff
@@ -503,7 +503,15 @@ def _list_outputs(self):
 
 
 # https://stackoverflow.com/a/4829130
-def search_files(prefix, outtypes):
-    return it.chain.from_iterable(
+def search_files(prefix, outtypes, search_crop):
```

Comment:
The tests are not verifying this functionality, and are not passing a `search_crop` argument. Please see `nipype/interfaces/tests/test_dcm2nii.py`.

Reply:
Test updated!